openassessment/workflow/test/test_api.py

```diff
@@ -361,6 +361,13 @@ def test_unable_to_load_api(self):
         with self.assertRaises(AssessmentWorkflowInternalError):
             workflow_api.create_workflow(submission['uuid'], ['self'], ON_INIT_PARAMS)
```

Comment:
Since we are using `transaction.atomic`, the changes the models in this method call get reverted. Previously, an `AssessmentWorkflow` was getting created. We need to do it explicitly now.